sdk/rust/src/lib.rs

```diff
@@ -8,6 +8,10 @@ pub mod outbound_http;
 /// Key/Value storage.
 pub mod key_value;
 
+/// Sqlite
+#[cfg(feature = "experimental-sqlite")]
```

Comment:
Is this for your testing, or do you intent that SDK users would need to turn the feature on in Cargo.toml until it stabilises?

Reply:
Oh sorry I see discussion of this in your ready-for-review comment.

Reply:
I had a look at the docs (https://developer.fermyon.com/spin/api-guides-overview) and we currently have several APIs that are described as "experimental" but to which don't require a specific opt-in by the developer.  (I don't present this as advocacy for either approach, just as s data point.)

Reply:
The current way of describing things as experimental but not putting any opt-in in front of the user is pretty dangerous, and I think we should move away from doing so. I plan on writing something up on how we can do experimental features in the future.